doc/source/ray-air/examples/torch_image_batch_pretrained.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pandas as pd
+from io import BytesIO
+from typing import List
+
+from PIL import Image
+from torchvision import transforms
+from torchvision.models import resnet18
+
+import ray
+from ray.air.util.tensor_extensions.pandas import TensorArray
+from ray.train.torch import to_air_checkpoint, TorchPredictor
+from ray.train.batch_predictor import BatchPredictor
+from ray.data.preprocessors import BatchMapper
+
+
+def convert_to_pandas(byte_item_list: List[bytes]) -> pd.DataFrame:
```

Comment:
add todo to remove this once ImageFolder merged ? I had the same for my CUJ PRs and we just need to make sure all can be search and replaced in one shot https://github.com/ray-project/ray/pull/26614

Reply:
done